### ISSUE TITLE
Vertically misaligned names in Person List

### ISSUE THREAD

**Issue by baskargopinath** [se-edu/addressbook-level3#214]
![image](https://github.com/se-edu/addressbook-level3/assets/73015364/f7b69547-58dd-4b0f-bd7b-37df0694362c)

**Comment by damithc** [comment]
Thanks for reporting this @baskargopinath

**Comment by baskargopinath** [comment]
no problem prof @damithc i am working on fixing this and will send another pr once the my existing pr is approved
